src/extension/src/Constants.py

```diff
@@ -27,6 +27,9 @@ def __iter__(self):
                     if item == self.__dict__[item]:
                         yield item
 
+    # Extension version (todo: move to a different file)
+    EXT_VERSION = "1.6.12"
+
```

Comment:
You can set this to 3.0.[%exec_sub_ver%] for anything that gets built using packager. There needs to be a better story here. Discuss in design requested previously...